tools/darwin/Support/GenerateMissingImages-tvos.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```

Comment:
Do not change this. Does not work on OSX Catalina

```
Support % ./GenerateMissingImages-tvos.py ../../../xbmc/platform/darwin/tvos/Assets.xcassets Assets LaunchImage
/Users/brent/osx-master/media/splash.jpg
  /Users/brent/osx-master/xbmc/platform/darwin/tvos/Assets.xcassets/LaunchImage.launchimage/splash@2x.jpg
/Users/brent/osx-master/xbmc/platform/darwin/tvos/Assets.xcassets/Assets.brandassets/topshelf_wide.imageset/image.png
  /Users/brent/osx-master/xbmc/platform/darwin/tvos/Assets.xcassets/Assets.brandassets/topshelf_wide.imageset/image@2x.png
Traceback (most recent call last):
  File "./GenerateMissingImages-tvos.py", line 28, in <module>
    for i in xrange(1, 5):
NameError: name 'xrange' is not defined
```

Reply:
Thanks for testing this! Yes xrange changes is trivial - I will walk through the files and fix the py2-specific issues.

Reply:
I fixed this issue. Please test it when you can.

Reply:
this change is ok now.